server/build/loaders/emit-file-loader.js

```diff
@@ -10,7 +11,14 @@ module.exports = function (content, sourceMap) {
   const opts = { context, content, regExp }
   const interpolatedName = loaderUtils.interpolateName(this, name, opts)
 
-  this.emitFile(interpolatedName, content, sourceMap)
+  const transpiled = transform(content, {
+    presets: [
+      'es2015'
+    ],
+    sourceMaps: 'both',
+    inputSourceMap: sourceMap
+  })
```

Comment:
I think it's cool if this transformation can be done as an option of this loaders, maybe by passing a function for changing `content` and `sourceMap`.